README.md

```diff
@@ -1 +1,32 @@
 # Railsdock
+
+A generic Dockerized Rails Development Environment
+
+## Setup
+
+* Clone the repo
+* Copy `env-example` to `.env`
+* Modify the `.env` to point to host directeries for each service on your machine. It defaults to the repo names at the same directory level as `railsdock`.
+> Note: only keys that are needed for build or general environment should be added here. All other environment variables should be placed in service specific `.env` files.
+* `docker-compose build` this will take awhile to build images for each service.
+* Bring up the specific services you need for example: `docker-compose up ruby postgres`. `docker-compose config --services` will list currently available services.
+
+## Misc Notes
+
+### Database
+`psql` is installed by default in the ruby container.
+
+* Import postgres database dump(This assumes a database has been created)
+
+```sh
```

Comment:
As a todo on this, what would you think about these commands in some form of `bin/setup` script?

Reply:
Yes, I'm down especially for initial app setup. A balance of scripts and plain old docker would probably worthwhile.